### PR TITLE
add solution items

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Review by CollinAlpert — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Lombok.NET
+~~~~# Lombok.NET
```
Comment (CollinAlpert) [inline README.md:1]: @SimonCropp Why this change? The README file looks weird now when displayed on the repository start page.

Comment (SimonCropp) [reply README.md:1]: I have no idea. want a pr to revert that bit?

Comment (CollinAlpert) [reply README.md:1]: No worries, I'll take care of it :)
